client/src/components/Map/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { withGoogleMap, GoogleMap, Marker } from "react-google-maps";
+
+import "./map.css";
+class Map extends Component {
+  render() {
+    // const markers = this.props.markers || [];
+    console.log(this.props.markers);
+    return (
+      <div className="map-style">
+        <GoogleMap
+          defaultZoom={this.props.zoom}
+          defaultCenter={this.props.center}
```

Comment:
1. Destructure `this.props` instead of retyping it in every line.
2. Remove commented code.
3. Remove console logs.
4. Add newline after last import.

Reply:
done